test/data/stages/systemd.unit.create/a.json

```diff
@@ -611,6 +611,13 @@
               "docs": true
             }
           }
+        },
```

Comment:
Out of curiosity, have you considered to add it to the unit test in https://github.com/osbuild/osbuild/blob/main/stages/test/test_systemd_unit_create.py#L95 instead of to the stage test? It seems (unless I miss something) to test the same and in the unit test we actually see the generated content, not just the hash. Or am I missing something?

Reply:
Just slipped my mind that we had it.  Added now and amended the stage to make them arrays in the process.